client/GlassServerControl.cs

```diff
@@ -772,9 +983,7 @@
         %obj.ctrl.setValue(getsubstr(%obj.ctrl.getValue(), 0, %parm));
       }
     }
-  }
-
-  if(%type $= "dropdown" || %type $= "playercount") {
+  } else if(%type $= "dropdown" || %type $= "playercount") {
```

Comment:
This will stop the Glass Server Preferences GUI from detecting any value changes from `num`, `string`, and `textarea` preference types and therefore stop the user from making changes to them e.g. the server name, server password, too-far distance, etc.

Reply:
Will just revert it in a follow up commit.